<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PUN-PREMG" ignored="0" name="Default test" tests="2" failures="0" timestamp="09 Jan 2018 12:15:03 GMT" time="38.367" errors="0">
  <testcase name="@AfterMethod LogForFail" time="0.0" classname="resources.base">
    <failure type="org.testng.TestNGException" message="
Can inject only one of &amp;lt;ITestContext, XmlTest, Method, Object[]&amp;gt; into a AfterMethod annotated LogForFail.
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Can inject only one of <ITestContext, XmlTest, Method, Object[]> into a AfterMethod annotated LogForFail.
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:244)
	at org.testng.internal.Parameters.createParameters(Parameters.java:172)
	at org.testng.internal.Parameters.createParameters(Parameters.java:458)
	at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:118)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:206)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:723)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- @AfterMethod LogForFail -->
  <testcase name="@AfterTest teardown" time="0.0" classname="resources.base">
    <skipped/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase name="LoginFlow" time="17.614" classname="MeridianPackages.Announcements"/>
  <testcase name="AnnouncementCreation" time="0.0" classname="MeridianPackages.Announcements">
    <skipped/>
  </testcase> <!-- AnnouncementCreation -->
</testsuite> <!-- Default test -->
